sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -66,7 +66,9 @@ def __call__(self, coef, intercept):
 
 
 class BaseSGD(SparseCoefMixin, BaseEstimator, metaclass=ABCMeta):
-    """Base class for SGD classification and regression."""
+    """Base class for SGD classification and regression.
+    .. version:: 0.21
```

Comment:
I don't think this is correct

Reply:
It doesn't appear in any change logs before 21. What should it be? For now, I'll delete the tag entirely.